doc/classes/NativeMenu.xml

```diff
@@ -5,6 +5,35 @@
 	</brief_description>
 	<description>
 		[NativeMenu] handles low-level access to the OS native global menu bar and popup menus.
+		[b]Note:[/b] This is low-level API, consider using [MenuBar] with [member MenuBar.prefer_global_menu] set to [code]true[/code], and [PopupMenu] with [member PopupMenu.prefer_native_menu] set to [code]true[/code].
+		To create a menu, use [method create_menu], add menu items using [code]add_*_item[/code] methods. To remove a menu, use [method free_menu].
+		[codeblock]
+		var menu
```

Comment:
I'm fine with this but how does anyone else feel about it, over the more explicit `var menu = null`.

Reply:
Seems redundant to me (at least for non-typed code, and all doc examples seems to be non-typed).